cmake/RunSimulation.cmake

```diff
@@ -100,32 +100,51 @@ if(BUILD_FORTRAN)
     execute_process(COMMAND ${CMAKE_COMMAND} -E copy "${IDF_PATH}" "${OUTPUT_DIR_PATH}/in.idf")
     execute_process(COMMAND ${CMAKE_COMMAND} -E copy "${EPW_PATH}" "${OUTPUT_DIR_PATH}/in.epw")
     execute_process(COMMAND ${CMAKE_COMMAND} -E copy_if_different "${PRODUCT_PATH}/Energy+.idd" "${OUTPUT_DIR_PATH}")
-    execute_process(COMMAND "${EXPANDOBJECTS_EXE}" WORKING_DIRECTORY "${OUTPUT_DIR_PATH}")
+    # This creates GHTIn.idf, and BasementGHTIn.idf IIF the GroundHeatTransfer:Control says to run it!
+    execute_process(COMMAND "${EXPANDOBJECTS_EXE}" WORKING_DIRECTORY "${OUTPUT_DIR_PATH}" COMMAND_ERROR_IS_FATAL ANY)
 
     if("${SLAB_RESULT}" GREATER -1)
-      # Copy files needed for Slab
-      file(COPY "${SOURCE_DIR}/idd/SlabGHT.idd" DESTINATION "${OUTPUT_DIR_PATH}")
-      # Find and run slab
-      find_program(SLAB_EXE Slab PATHS "${PRODUCT_PATH}" NO_DEFAULT_PATH NO_CMAKE_ENVIRONMENT_PATH NO_CMAKE_PATH NO_SYSTEM_ENVIRONMENT_PATH
-                                                         NO_CMAKE_SYSTEM_PATH NO_CMAKE_FIND_ROOT_PATH)
-      message("Executing Slab from ${SLAB_EXE}")
-      execute_process(COMMAND "${SLAB_EXE}" WORKING_DIRECTORY "${OUTPUT_DIR_PATH}")
-      # Then copy slab results into the expanded file
-      file(READ "${OUTPUT_DIR_PATH}/SLABSurfaceTemps.TXT" SLAB_CONTENTS)
-      file(APPEND "${OUTPUT_DIR_PATH}/expanded.idf" "${SLAB_CONTENTS}")
+      if(NOT EXISTS "${OUTPUT_DIR_PATH}/GHTin.idf")
```

Comment:
Could understand why decent CI Linux was not finding it, while It was working on mac...

I'm always saying "beware that macOS/Windows have case-insensitive filesystem while Linux is case-sensitive", and I still made that mistake.